mongodb.js

```diff
@@ -0,0 +1,102 @@
+const { MongoClient, Timestamp } = require("mongodb");
+
+module.exports = { db_connect, db_setup, new_print_job, new_workflow, new_workflow_step, new_simulation_report };
+
+
+async function db_connect(){
+	const uri = "mongodb://localhost:27017/HP";
+	const client = new MongoClient(uri);
+	console.log("Connecting to MongoDB...");
+	await client.connect();
+	console.log("Database connected");
+	const database = client.db('HP');
+	return [client, database];
+}
+
+
+// DELETES all existing collections, recreates them, then inserts
+// some dummy data
+async function db_setup(database){
+	//Drop all collections
+	const old_collections = await database.listCollections().toArray();
+	Promise.all(old_collections.map((c) => database.collection(c.name).drop()));
+
+	//Create collections
+	const collections = ["PrintJob", "Workflow", "WorkflowStep", "SimulationReport"];
+	Promise.all(collections.map((c) => database.createCollection(c)));
+
+	//Insert dummy data
+	const pj_id = await new_print_job(database, "PrintJob 1", 5, ["RP 1"]);
```

Comment:
This dummy data is useful for testing purposes, but should not be present in the actual function. We need to create a separate test file that will insert dummy data.